modules/core/src/test/java/org/apache/ignite/testframework/GridTestUtils.java

```diff
@@ -2129,7 +2129,22 @@ public static void addTestIfNeeded(@NotNull final List<Class<?>> suite, @NotNull
      * @return Random string object.
      */
     public static String randomString(Random rnd, int maxLen) {
-        int len = rnd.nextInt(maxLen);
+        return randomString(rnd, 0, maxLen);
+    }
+
+    /**
+     * Generate random alphabetical string.
+     *
+     * @param rnd Random object.
+     * @param maxLen Maximal length of string
+     * @param minLen Minimum length of string
```

Comment:
I suggest swapping the parameters maxLen and minLen so that they match the signature of the method. Also trailing dot is missing.

Reply:
fixed